tests/unit/providers/test_ollama_provider.py

```
@@ -128,6 +128,69 @@ async def empty_async_iter() -> AsyncIterator[None]:
+            CompletionParams(
+                model_id="llama3.1",
+                messages=[{"role": "user", "content": "find three fixtures"}],
+                stream=True,
+            ),
+        )
+
+        indices = []
+        async for chunk in result:  # type: ignore[union-attr]
+            for choice in chunk.choices:
+                for tool_call in choice.delta.tool_calls or []:
+                    indices.append(tool_call.index)
```

Ruff (0.15.20)

[warning] Use `list.extend` to create a transformed list

(PERF401)